lib/insights_cloud/async/insights_scheduled_sync.rb

```diff
@@ -0,0 +1,25 @@
+require 'rest-client'
+
+module InsightsCloud
+  module Async
+    class InsightsScheduledSync < ::ApplicationJob
```

Comment:
Where is the first execution of this scheduled from?

Reply:
Nice catch, fixed: https://github.com/theforeman/foreman_rh_cloud/pull/277/files#diff-8baf49eae5c74fe89761eae48984cfb3R74